File: libgomp/testsuite/libgomp.oacc-c-c++-common/data-already-7.c
/* { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } } */

#include <stdio.h>
#include <openacc.h>

int
main (int argc, char *argv[])
{
  int i;

#pragma acc enter data create (i)
  fprintf (stderr, "CheCKpOInT\n");
  acc_create (&i, sizeof i);

  return 0;
}

/* { dg-output "CheCKpOInT(\n|\r\n|\r).*" } */
/* { dg-output "already mapped to" } */
/* { dg-shouldfail "" } */

File: libgomp/testsuite/libgomp.oacc-fortran/data-already-7.f
! { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } }

      IMPLICIT NONE
      INCLUDE "openacc_lib.h"

      INTEGER I

!$ACC ENTER DATA CREATE (I)
      WRITE(0, *) "CheCKpOInT"
      CALL ACC_CREATE (I)

      END

! { dg-output "CheCKpOInT(\n|\r\n|\r).*" }
